docker-compose-mysql.yml

```diff
@@ -1,37 +1,17 @@
-mysql:
+# Run MySQL instead of Cassandra
+storage:
   image: openzipkin/zipkin-mysql:1.37.0
   ports:
     - 3306:3306
-query:
-  image: openzipkin/zipkin-query:1.37.0
+
+# Switch storage type to MySQL
+collector:
```

Comment:
this was left out of mysql on purpose, to show that you don't need to run collector proces to accept from http. Also let's please not encourage any more use of scribe.